components/automate-ui/src/app/pages/team-details/team-details.component.ts

```diff
@@ -52,9 +59,25 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
   public removeText = 'Remove User';
 
   constructor(private store: Store<NgrxStateAtom>,
-    private fb: FormBuilder
+    public fb: FormBuilder,
+    private router: Router
   ) {
-    this.createForms(this.fb);
+    this.updateNameForm = fb.group({
+      // Must stay in sync with error checks in team-details.component.html
+      name: ['Loading...', Validators.required]
```

Comment:
Needs to also check for all whitespace. (This matches the suggestion I made on the template side.)
```suggestion
      name: ['Loading...', [Validators.required, Validators.pattern(Regex.patterns.NON_BLANK)]]
```